vehicle/OVMS.V3/main/ovms_metrics.h

```diff
@@ -1110,7 +1111,7 @@ class OvmsMetrics
   };
 
 extern OvmsMetrics MyMetrics;
-extern UserMetricMap MyUserMetrics;
+extern UserUnitConfigMap MyUserUnitConf;
```

Comment:
I'd rather remove `User` from both the class and the instance name, and then not abbreviate `Config`. `User` is redundant, as unit configuration implies user configuration. `Conf` as an abbreviation for `Config` is somewhat unusual, as `Config` is already an abbreviation. But that's more a personal style choice.

Reply:
Consider it done.  Conf is an abbreviation I've use but I've no arguments against making it Config. Especially when making it 'MyUnitConfig' .. and you're right,  'My' makes 'User' redundant.